docs/mute.md

```diff
@@ -43,7 +43,20 @@ Note that the assertion is verified at the end of the program, so it doesn't sto
 You can add many ```TIMEOUT``` assertion in the same program, but, obviously, just the one with the shortest timeout is considered.
 
 There isn't a mandatory position to insert the ```TIMEOUT``` annotation line, that, as other ```MUTE``` annotations, are just considered as comments by the standard interpreter engine.
- 
+
+### Failing assertions 
+Like in other frameworks for unit testing, there is an assertion that always fails: 
```

Comment:
We could add a newline between the title and the paragrah

Reply:
OK